src/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```diff
@@ -40,6 +43,12 @@
     <Compile Include="$(CommonTestPath)\System\Net\Capability.Security.cs">
       <Link>Common\System\Net\Capability.Security.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Net\Capability.Security.Windows.cs" Condition="'$(TargetsWindows)' == 'true'">
+      <Link>Common\System\Net\Capability.Security.Windows.cs</Link>
+    </Compile>
```

Comment:
This file appears to be missing.

Reply:
Thanks.  I'm still learning about command line git versus the easy Git Desktop for Windows which moves all files into a commit.